server/app.js

```diff
@@ -3,6 +3,8 @@ const express = require("express");
 const { join } = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
+const mongoose = require('mongoose');
+require('dotenv').config()
```

Comment:
I noticed you are missing a semicolon - meaning you probably don't have prettier.io set up on your project - I would recommend to have that set up so we have a consistent styling across the board. 

We shouldn't have to require('dotenv') here actually because I believe we already do it in the www/bin, but I could be wrong

Reply:
I installed prettier as a plugin. But it didn't work as you pointed out. I will try if there is some configuration I am missing. I will remove the require('dotenv'). It is already in www/bin.

Reply:
I removed require('dotenv'). Please review